community/contributors.md

```diff
@@ -0,0 +1,165 @@
+
+With either path you become a "provisional core developer". As such you will need to show consistent contributions of code and/or specifications to the project. This can be in the form of pull requests that get merged into master. Or it can be in the form of technical specification, system architecture and related artifacts that guide the development activities of others.
+
+All provisional core developers that focus on code development (over standards) must meet with the existing core developers and demonstrate they are capable of the following:
+
+* Running the project locally
+* Using the testing framework
+* Explaining the components of the system architecture
+* Walking through the code and explain the baseline process
+
+Once the provisional core developer demonstrates their capabilities, the existing core developers will vote during the next scheduled core developer meeting to give the prospect full core developer status. Members must vote with 2/3rds majority to add a core developer. Voting that results in a tie or potentially other issue will be brought to the TSC for review.
+
+## What is expected of core developers? <a href="#what-is-expected-of-maintainers" id="what-is-expected-of-maintainers"></a>
+
+In general, a core developer needs to:
+
+* be an expert in one or more fields related to the project
+* be an expert in finding and engaging the advice of other experts
+* show commitment over time with multiple PRs merged
+* be reliable in completing issues to which they have been assigned
+* attend the weekly core developers meetings (with occasional absences allowed)
+* demonstrate competency in software development or specification writing
+* follow the project style and testing guidelines
+* have a high degree of understanding of the project architecture
+* be welcoming to others in the community who are using the project
+* contribute in ways that substantially improve the quality of the project and the experience of people who use it
+* follow branch, PR, and code style conventions
+
+## How core developers organize? <a href="#how-maintainers-organize" id="how-maintainers-organize"></a>
+
+### Slack <a href="#slack" id="slack"></a>
+
+Core Developers meet and discuss issues virtually via the #maintainers slack room in the [baseline slack](https://ethereum-baseline.slack.com).
+
+### Regular Meetings <a href="#weekly-meetings" id="weekly-meetings"></a>
+
+There are weekly _Core Developers meetings_ where members can discuss plans and issues related to the project, updates, release planning, and other related topics. Anyone may attend these meetings, but the primary participants are core developers. Core developers are required to produce meeting summaries and document decisions.&#x20;
+
+Meeting signups are maintained on the "Join the Community" page on https://baseline-protocol.org.
+
+## How to stop being a core developer? <a href="#how-to-stop-being-a-maintainer" id="how-to-stop-being-a-maintainer"></a>
```

Comment:
This seems to be a misleading statement. I mean we don't want them to do any of these like being disrespectful to others, etc. 

`Suggestion : `
How to stop being a core developer? 
You notify the other core developers  you would like to relinquish your core developer status.

What can take away your Core Dev status?
* You stop reviewing PR's, responding to messages, answering emails, and/or generally ghost the project.
* You are disrespectful towards anyone in the community and/or involved in the project.
* You are disruptive to the general process of maintaining the project, meetings, discussions, issues, or other.